app/assets/stylesheets/administrate/base/_typography.scss

```diff
@@ -30,15 +30,15 @@ a,
 ) {
   color: $action-color;
   text-decoration-skip-ink: auto;
-  transition: color $base-duration $base-timing;
+  transition: color var(--base-duration) var(--base-timing);
 
   &:hover {
-    color: mix($black, $action-color, 25%);
+    color: var(--action-color-active);
```

Comment:
This changed ever so slightly. There were two selectors that used `$action-color` as their base. This one mixes in 25% `$black` and `&:not(.link):hover` in `buttons.scss` mixes in 20% `$black`. I went with the 20% in both places so we only had one "it's like `$action-color` but tinted" variable because button backgrounds are more prominant than text hovers.